src/main/java/org/ibp/api/java/impl/middleware/germplasm/GermplasmExcelTemplateExportServiceImpl.java

```diff
@@ -144,93 +159,27 @@ private File generateTemplateFile(final String fileNamePath, final String cropNa
-		cell.setCellStyle(this.sheetStylesMap.get(ExcelCellStyle.HEADING_STYLE_BLUE));
-		cell.setCellValue(this.getMessageSource().getMessage("export.germplasm.list.template.stock.id.column", null, locale));
-
-		cell = row.createCell(GermplasmExcelTemplateExportServiceImpl.OBSERVATION_SHEET_GUID_COLUMN_INDEX, CellType.STRING);
-		cell.setCellStyle(this.sheetStylesMap.get(ExcelCellStyle.HEADING_STYLE_ORANGE));
-		cell.setCellValue(this.getMessageSource().getMessage("export.germplasm.list.template.guid.column", null, locale));
-
-		observationSheet.setColumnWidth(GermplasmExcelTemplateExportServiceImpl.OBSERVATION_SHEET_ENTRY_NO_COLUMN_INDEX, 13 * 250);
-		observationSheet.setColumnWidth(GermplasmExcelTemplateExportServiceImpl.OBSERVATION_SHEET_LNAME_COLUMN_INDEX, 13 * 250);
-		observationSheet.setColumnWidth(GermplasmExcelTemplateExportServiceImpl.OBSERVATION_SHEET_DRVNM_COLUMN_INDEX, 13 * 250);
-		observationSheet.setColumnWidth(GermplasmExcelTemplateExportServiceImpl.OBSERVATION_SHEET_PREFERRED_NAME_COLUMN_INDEX, 20 * 250);
-		observationSheet.setColumnWidth(GermplasmExcelTemplateExportServiceImpl.OBSERVATION_SHEET_ENTRY_CODE_COLUMN_INDEX, 16 * 250);
-		observationSheet.setColumnWidth(GermplasmExcelTemplateExportServiceImpl.OBSERVATION_SHEET_LOCATION_ABBR_COLUMN_INDEX, 20 * 250);
-		observationSheet.setColumnWidth(GermplasmExcelTemplateExportServiceImpl.OBSERVATION_SHEET_REFERENCE_COLUMN_INDEX, 13 * 250);
-		observationSheet.setColumnWidth(GermplasmExcelTemplateExportServiceImpl.OBSERVATION_SHEET_CREATION_DATE_COLUMN_INDEX, 18 * 250);
-		observationSheet.setColumnWidth(GermplasmExcelTemplateExportServiceImpl.OBSERVATION_SHEET_BREEDING_METHOD_COLUMN_INDEX, 22 * 250);
-		observationSheet.setColumnWidth(GermplasmExcelTemplateExportServiceImpl.OBSERVATION_SHEET_NOTES_COLUMN_INDEX, 13 * 250);
-		observationSheet.setColumnWidth(GermplasmExcelTemplateExportServiceImpl.OBSERVATION_SHEET_STORAGE_LOCATION_ABBR_COLUMN_INDEX, 28 * 250);
-		observationSheet.setColumnWidth(GermplasmExcelTemplateExportServiceImpl.OBSERVATION_SHEET_UNITS_COLUMN_INDEX, 13 * 250);
-		observationSheet.setColumnWidth(GermplasmExcelTemplateExportServiceImpl.OBSERVATION_SHEET_AMOUNT_COLUMN_INDEX, 13 * 250);
-		observationSheet.setColumnWidth(GermplasmExcelTemplateExportServiceImpl.OBSERVATION_SHEET_STOCK_ID_COLUMN_INDEX, 13 * 250);
-		observationSheet.setColumnWidth(GermplasmExcelTemplateExportServiceImpl.OBSERVATION_SHEET_GUID_COLUMN_INDEX, 13 * 250);
+		final HSSFRow row = observationSheet.createRow(0);
+
+		final List<String> headers = new ArrayList<>(isUpdateFormat ? IMPORT_HEADERS_FOR_UPDATE : IMPORT_HEADERS);
+		final ListIterator<String> iterator = headers.listIterator();
+		while (iterator.hasNext()) {
+			final int index = iterator.nextIndex();
+			final String header = iterator.next();
+			HSSFCell cell = row.createCell(index, CellType.STRING);
+			cell.setCellStyle(this.sheetStylesMap.get(ExcelCellStyle.HEADING_STYLE_YELLOW));
```

Comment:
You are setting all cell colors as yellow. However in the import germplasm, some cells are blue, pale blue, orange. 
Is this cleared with @clarysabel ?

Reply:
I didn't notice that some of the headers have different colors. I'll just revert it back.

Reply:
You can compare with the ones in the import germplasm template. We have already reviewed the colors with Mariano using that file